x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -310,9 +310,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       ...(isRuleRegistryEnabled ? racRuleTypes : []),
```

Comment:
Based on this remaining conditional on `isRuleRegistryEnabled`, even if the sub-feature privilege is shown, RAC rule types will not be included unless the feature flag is enabled.

Is this expected?

Reply:
Yes! That's expected. For 7.15 we are not shipping with rule registry rules.